tests/integration/cattletest/core/common_fixtures.py

```diff
@@ -69,12 +68,11 @@ def accounts():
     result = {}
     admin_client = _admin_client()
     for user_name in ['admin', 'agent', 'user', 'agentRegister', 'test',
-                      'readAdmin', 'token']:
+                      'readAdmin', 'internalTest']:
```

Comment:
Note that internalTest appears to be replacing token in the diff. Discussed with pranav. He may need to add token back in. Though if it isnt breaking tests, maybe it isnt needed and it can be dropped. Thoughts @ibuildthecloud